piece_of_cake.py

```diff
@@ -1,7 +1,8 @@
 def get_recipe_price(prices, optionals=[], **ingredients):
-    total = 0
-    for ing in ingredients:
-        if ing not in optionals:
-            total += ingredients.get(ing)*prices.get(ing)
-    return total//100
-
+    """
+    receives as parameter a dictionary of ingredients : price for 100gr ,
```

Comment:
the documentation isn't written according to the standard where each parameter is described in a separate line